state/protocol/events.go

```diff
@@ -84,10 +84,24 @@ type Consumer interface {
 	// NOTE: Only called once the phase transition has been finalized.
 	EpochCommittedPhaseStarted(currentEpochCounter uint64, first *flow.Header)
 
-	// EpochEmergencyFallbackTriggered is called when epoch fallback mode [EFM] is triggered.
-	// Since EFM is a permanent, spork-scoped state, this event is triggered only once.
-	// After this event is triggered, no further epoch transitions will occur,
-	// no further epoch phase transitions will occur, and no further epoch-related
-	// related protocol events (the events defined in this interface) will be emitted.
-	EpochEmergencyFallbackTriggered()
+	// EpochFallbackModeTriggered is called when epoch fallback mode [EFM] is triggered.
```

Comment:
consistent capitalization of acronym and the text it represents 
```suggestion
	// EpochFallbackModeTriggered is called when Epoch Fallback Mode [EFM] is triggered.
```